cmd/statusd/utils.go

```diff
@@ -835,7 +837,7 @@ func testCompleteMultipleQueuedTransactions(t *testing.T) bool {
 
 	//  this call blocks, and should return when DiscardQueuedTransaction() for a given tx id is called
 	sendTx := func() {
-		txHashCheck, err := statusAPI.SendTransaction(nil, common.SendTxArgs{
+		txHashCheck, err := statusAPI.SendTransaction(context.TODO(), common.SendTxArgs{
```

Comment:
Good catch! By the way, why `TODO` instead of `Background`?

Reply:
@tiabc Both of them are empty context.  
context.Background - need empty context. 
context.TODO - it's unclear which Context to use.
It helps us to find all unclear context usage later.

@divan most of the affected files are related to tests =(

Reply:
But here we have a test. Do we want `context.TODO` to draw our attention here? I guess `context.Background` is perfectly fine or let's use `context.WithTimeout` to prevent from running forever.

Reply:
Scope of this issue could grow fast. I use it, because the issue is fix linters warnings. I think `Adding context.WithTimeout to tests` is separated issue and we could do it, if we need.